giskard/client/giskard_client.py

```diff
@@ -74,6 +73,15 @@ def __call__(self, r):
         return r
 
 
+def _limit_str_size(json, field, limit=255):
+    if field not in json or not isinstance(json[field], str):
+        return
+
+    if len(json[field]) > limit:
```

Comment:
I tested emoji in the name.

In Python 3, `len`  counts the number of utf-8 characters:

```
>>> len("123✋56")
6
>>> len("123✋🔥56")
7
```

while in the Java backend and database, the case could be different:

<img width="713" alt="image" src="https://github.com/Giskard-AI/giskard/assets/8311300/e8e570df-a9f2-48ac-ba4f-0ec3d70bd139">

<img width="613" alt="image" src="https://github.com/Giskard-AI/giskard/assets/8311300/030e11eb-4dab-4fa0-915d-98e651bc5e8a">

Could we check the number of utf-16 characters instead?
Ref: https://docs.oracle.com/javase/8/docs/api/java/nio/charset/Charset.html#:~:text=The%20native%20character%20encoding%20of,chars)%20and%20sequences%20of%20bytes.

Reply:
In addition, the case could also be different in PostgreSQL in our production env. We may have to check deeper the encoding used in production.

Reply:
Seems that we use the EN_UTF-8 local in our env since we use `12-alpine` without LANG variable: https://github.com/docker-library/postgres/blob/038c4c577a3c58dddf9ec2ccaa643009b8ba414b/12/alpine3.18/Dockerfile

Reply:
I fixed the truncate logic

Reply:
Thanks very much!

To confirm:
1. we have utf-8 between client/front-end and backend.
2. In Java backend, they are stored with utf-16 in `String`.
3. And to the DB, we have utf-16 for H2 and utf-8 for PostgreSQL.

I am not sure where the length is validated: is it in the DB or in the Hibernate? Could the requirement be different?